components/brave_wallet_ui/panel/async/wallet_panel_async_handler.ts

```diff
@@ -616,6 +649,13 @@ handler.on(WalletActions.initialize.type, async (store) => {
       store.dispatch(PanelActions.signMessage(signMessageRequests))
       return
     }
+
+    const signMessageErrors = await getPendingSignMessageErrors()
```

Comment:
Just a thought, should pending sign message errors be shown before pending sign message requests (force user to acknowledge the error before proceeding)? If not the user could dismiss panel without tapping `Close` and tap `Sign in with Ethereum` button again and then I believe they'd see the Sign Message Error after the successful SIWE request.
cc @Douglashdaniel @josheleonard

Reply:
Good thought, moved above `Sign Data` in the Panel `Container`